profiles/kentik_snmp/gigamon/gigamon.yml

```diff
@@ -0,0 +1,33 @@
+---
+extends:
+  - if-mib.yml
+  - system-mib.yml
+
+provider: kentik-switch
+
+sysobjectid:
+  - 1.3.6.1.4.1.26866.3.6 # GigaVUE-HC2
+
+metrics:
+  - MIB: GIGAMON-SNMP-MIB
+    symbol:
+      # Displays the average cpu load(in % per 30 seconds) in CPU utilization of the system.
+      name: gigamonCpuUtilization
+      OID: 1.3.6.1.4.1.26866.2.18.4
```

Comment:
From the way this is set up here I believe these OID's are scalars and should end with .0

Can you confirm for me that in your walk they do or don't have .0 as the last segment? @araddas

Reply:
@Mesverrum you are correct, updated the oids.